Task3 layout

denisbaranchik opened the pull request:
layout created

MaksimViarenich commented:
You did a good job! I understand that it's not easy to write a good html+css.
We can discuss on the lesson the most common mistakes and if you have any questions you are welcome. You. have too much CSS, it can be decreased twice

MaksimViarenich commented:
You can set in IntelliJ Code style for your HTML/CSS and the fix code style by `Ctrl+Shift+L`
<img width="1132" alt="Снимок экрана 2020-03-02 в 22 33 58" src="https://user-images.githubusercontent.com/60959134/75710707-0c756b80-5cd6-11ea-8f2c-ec57bab3cf52.png">

denisbaranchik commented:
Thanks! Let's discuss everything at the lesson first

denisbaranchik commented:
It seems i fixed everything you've mentioned:

-Get rid of redundant divs in html
-Get rid of style in html
-Made better checkboxes
-Made buttons with <button>
-Fixed divs overflow
-Updated css length and style

MaksimViarenich commented on UI/index.html:
```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html>
+
```
Minor. Remove one empty line

MaksimViarenich commented on UI/index.html:
```diff
@@ -0,0 +1,186 @@
+
+                <div class="container">
+                    <input type="checkbox" id="checkbox-date">
+                    <label for="chackbox-date" class="text-style-1 checkbox-label">Date</label>
```
checkbox-date, not chackbox-date

MaksimViarenich commented on UI/index.html:
```diff
@@ -0,0 +1,186 @@
+                    <label for="chackbox-date" class="text-style-1 checkbox-label">Date</label>
+                </div>
+
+                <input type="text" id="ffrom" name="ffrom" placeholder="From">
```
Bad names for id and name

MaksimViarenich commented on UI/index.html:
```diff
@@ -0,0 +1,186 @@
+                </div>
+
+                <input type="text" id="ffrom" name="ffrom" placeholder="From">
+                <input type="text" id="fto" name="fto" placeholder="To">
```
Bad names

MaksimViarenich replied:
For dates use type date, not text

MaksimViarenich commented:
Merge, please, PR)

denisbaranchik commented:
I thought you should approve first, but if you say so...
